cinema/models.py

```diff
@@ -84,23 +90,27 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
-    def clean(self):
-        for ticket_attr_value, ticket_attr_name, cinema_hall_attr_name in [
-            (self.row, "row", "rows"),
-            (self.seat, "seat", "seats_in_row"),
-        ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
+    @staticmethod
+    def validate_row_and_seat(
+            row: int, num_row: int,
+            seat: int, num_seat: int,
+            error_to_raise
+    ) -> None:
+        if not (1 <= row <= num_row):
+            raise error_to_raise(
+                {"row": f"Row must be in range (1, {num_row})."}
+            )
+        if not (1 <= seat <= num_seat):
+            raise error_to_raise(
+                {"seat": f"Seat must be in range (1, {num_seat})."}
             )
-            if not (1 <= ticket_attr_value <= count_attrs):
-                raise ValidationError(
-                    {
-                        ticket_attr_name: f"{ticket_attr_name} "
-                        f"number must be in available range: "
-                        f"(1, {cinema_hall_attr_name}): "
-                        f"(1, {count_attrs})"
-                    }
-                )
+
+    def clean(self):
+        Ticket.validate_row_and_seat(
+            self.row, self.movie_session.cinema_hall.rows,
+            self.seat, self.movie_session.cinema_hall.seats_in_row,
+            ValueError
```

Comment:
In the `clean` method of the `Ticket` model, `ValueError` is used as the error to raise. It would be more appropriate to use `ValidationError` from `django.core.exceptions` for model validation errors. This ensures that the error is handled correctly by Django's validation framework.